src/fides/api/ctl/migrations/versions/d65bbc647083_adds_gpc_info_to_consent_table.py

```diff
@@ -0,0 +1,34 @@
+"""adds GPC info to consent table
+
+Revision ID: d65bbc647083
+Revises: 8e198eb13802
+Create Date: 2023-02-15 13:57:36.159161
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd65bbc647083'
+down_revision = '8e198eb13802'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('consent', sa.Column('has_gpc_flag', sa.Boolean(), nullable=True))
+    op.add_column('consent', sa.Column('conflicts_with_gpc', sa.Boolean(), nullable=True))
+    op.execute("UPDATE consent SET has_gpc_flag = false")
+    op.alter_column('consent', 'has_gpc_flag', server_default="f", default=False, nullable=False)
+    op.execute("UPDATE consent SET conflicts_with_gpc = false")
+    op.alter_column('consent', 'conflicts_with_gpc', server_default="f", default=False, nullable=False)
```

Comment:
Just double checking, is `f` correct for `server_default`? It gets set to `false` in the query.